### PR TITLE
Add Webpack setup

### LINKED ISSUE #62: Pick a JS bundler
We need to decide on which bundler we want to use, then
- add it to the shell to use it manually when needed
- extend the derivations (e.g. `bundlePursProject`) in `nix/lib.nix`

We've discussed either using Parcel or Webpack

### PR DESCRIPTION
**Pull request by ngua**
Closes #62

Adds support for building/bundling the project with Webpack, since Parcel doesn't support WASM yet. There are new scripts to run the dev server or build the project

There are some placeholders for entry points at the moment

### CODE REVIEW

**Review by MatthewCroughan — APPROVED** [review]
I would say the Nix complexity of this repository is quickly getting out of hand. And since I haven't been reviewing the Nix changes until the CODEOWNERS file was added, I can't give any constructive feedback.

The Nix changes in this commit however, are fine.

**Review by vvtran — COMMENTED** [review]
Just one comment.

File: examples/nami/Simple.purs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Examples.Nami.Simple where
```
[the remark this answers is not in the document]
Comment (ngua) [reply examples/nami/Simple.purs:1]: Ah yes, I'll add that

**Review by nixinator — APPROVED** [review]

**Review by nixinator — APPROVED** [review]
Second what @MatthewCroughan  

dream2nix might be a good way of reducing complexity here, especially with npm in nix

**Review by klntsky — COMMENTED** [review]
Should README be updated?

**Comment by ngua** [comment]
@klntsky Good call, I'll add something

**Pull request merged**